python/python/lance/lance/__init__.pyi

```diff
@@ -32,6 +32,14 @@ class CompactionMetrics:
     fragments_added: int
     files_removed: int
     files_added: int
+    
+    def __repr__(self):
+        return f"""
+        Fragments Removed: {self.fragments_removed}
+        Fragments Added: {self.fragments_added}
+        Files Removed: {self.files_removed}
+        Files Added: {self.files_added}
+    """
```

Comment:
I don't think function implementations are meant to be put into `.pyi` files, which are type stubs. 

`CompactionMetrics` is implemented in Rust here: 

https://github.com/lancedb/lance/blob/1b8adb56f209eb0690c0a2e95ea06896188e656a/python/src/dataset/optimize.rs#L78

Could you implement the method there in Rust? Here is an example Rust implementation of a `__repr__` method:

https://github.com/lancedb/lance/blob/1b8adb56f209eb0690c0a2e95ea06896188e656a/python/src/dataset/optimize.rs#L113-L119

Reply:
Thanks for getting back, I really appreciate the support. This is my first time contributing to open-source project. It'll try to implement this and get back.